packages/@lightningjs/ui-components/docs/ThemingComponentConfig.stories.mdx

```diff
@@ -0,0 +1,101 @@
+<!--
+  Copyright 2023 Comcast Cable Communications Management, LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  SPDX-License-Identifier: Apache-2.0
+-->
+
+
+import { Meta, Description } from '@storybook/addon-docs';
+
+<Meta title="Docs / Theming / Component Config" />
+
+# Component Config
+
+The `componentConfig` object can take in key value pairs of component names to new objects. These objects contain theme values that can override both the standard component properties and the predetermined style properties of the given component type.
+
+## Style
+
+The `style` object can be used to override the predetermined style properties of the given component type.
+
+For example:
+```js
+componentConfig: {
+  ProgressBar: {
+    style: { radius: 20 }
+  }
+}
+```
+
+## Tones
+
+For a deeper dive on `tones`, see ["Tones"](../?path=/story/docs-theming-tones--page).
+
+The `tone` property expects a string which will determine the default color palette to apply to a component. It should match one of the following:
```

Comment:
Should we instead say something like "Unless you have created your own tone(s), it should match one of the following default options:"? Just to make it clear that those aren't always the only ones available?